pysat/tests/test_utils.py

```diff
@@ -192,6 +184,38 @@ def test_scale_units_both_bad(self):
         assert str(verr).find('unknown units') > 0
 
 
+class TestListify():
+    def setup(self):
+        """Runs before every method to create a clean testing setup."""
```

Comment:
```suggestion
        """Runs before every method to create a clean testing setup."""
        return
```

Reply:
This change was included so not sure why it still shows up here.